subsys/bluetooth/host/mesh/settings.c

```diff
@@ -159,31 +174,25 @@ static int net_set(int argc, char **argv, char *val)
 	return 0;
 }
 
-static int iv_set(int argc, char **argv, char *val)
+static int iv_set(int argc, char **argv, void *val_ctx)
 {
 	struct iv_val iv;
-	int len, err;
+	int err;
 
-	BT_DBG("val %s", val ? val : "(null)");
+	if (settings_val_get_len_cb(val_ctx) == 0) {
+		BT_DBG("val (null)");
```

Comment:
I realize this is just copied from the original log string, but more intuitive would perhaps be "IV deleted" or something like that - isn't that what this branch is about?